policyengine_us/parameters/gov/states/al/tax/income/deductions/itemized/sources.yaml

```diff
@@ -12,6 +12,8 @@ values:
     - al_casualty_loss_deduction # Section (a), (6)
     - al_misc_deduction # Section (a), (23)
     - al_interest_deduction # Section (a), (2)
+  2023-01-01:
```

Comment:
add of the exemptions from above has to also be included in the list! Not just the new one